server/db/seederMySql.js

```diff
@@ -0,0 +1,123 @@
+const fs = require('fs');
+const path = require('path');
+const os = require('os');
+const loremIpsum = require("lorem-ipsum").loremIpsum;
+const mysql = require('mysql');
+//const csv = require('fast-csv');
+var csv = require('csv-parser')
+
+
+const mysqlConfig = {
+  host: 'localhost',
+  user: 'glkapitan',
+  password: 'student',

```

Comment:
You should store this in an environment variable. Passwords shouldn't be committed to git. In this case it probably doesn't matter, but it would be really really bad if you committed your AWS keys.